common/Projectile.js

```diff
@@ -1,7 +1,7 @@
 import nengi from "nengi";
 
 class Projectile {
-    constructor(x, y, rotation) {
+    constructor(x, y, rotation, owner) {
```

Comment:
nitpick: as a reader of this code, the naming of this parameter -- owner -- implies that it is a **memory reference** to a fully formed thing, i.e. a memory ref to the actual player entity that owns it.

in reality, this is only a number - you are passing the owners **id**! so a better name that I'll request out for something like this in a proper codebase would be `ownerId`